chainer/functions/loss/negative_sampling.py

```diff
@@ -314,8 +317,10 @@ def backward(self, indexes, grad_outputs):
         return ret
 
 
-def negative_sampling(x, t, W, sampler, sample_size, reduce='sum'):
-    """Negative sampling loss function.
+def negative_sampling(x, t, W, sampler, sample_size, reduce='sum', **kwargs):
```

Comment:
Why is `return_samples` keyword only?

Reply:
To be conservative: We don't have to define fixed ordering. It's benefitial for both users and us.

* For users: Forces easier-to-understand keyword notation.
* For us: Easier to deprecate/add options in the future.

Reply:
OK.